docs/plugins/transform-react-inline-elements.md

```diff
@@ -10,14 +10,16 @@ Replaces the `React.createElement` function with one that is more optimized for
 
 <blockquote class="babel-callout babel-callout-info">
   <p>
-    Note that this requires the global polyfill due to React's use of ES6 <code>Symbol</code> when validating React Elements.
-    If <code>Symbol</code> is not present in the browser, your application will fail to render, as Babel will polyfill <code>Symbol</code>
-    but React will not have access to that polyfill.
+    Note that this might require the Symbol's global polyfill due to React's use of ES6 <code>Symbol</code> when validating React Elements.
```

Comment:
Yeah, and to your point in the original issue, they probably belong in the package's README so that users viewing docs in npm/GitHub and the website alike can get the info. At least it will make it less confusing where the source is :)

/cc: @xtuc @hzoo